### PR TITLE
Feat/e2e pipeline

### PR DESCRIPTION
**Pull request by JonathanConn**
WIP 

# Changes
- feat: Added e2e pipeline

### CODE REVIEW

File: .github/workflows/pr_e2e_tests.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Pull request E2E tests
+
```
Comment (arein) [inline .github/workflows/pr_e2e_tests.yml:2]: why are you ignoring my previous PR feedback here?

Please either say you disagree with the feedback or agree with it but don't ignore it.

File: .github/workflows/pr_e2e_tests.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+on:
+  pull_request:
+    branches:
+      - V2
```
Comment (arein) [inline .github/workflows/pr_e2e_tests.yml:6]: feels like a regression to the previous iteration where you also ran upon `push`

**Review by arein — CHANGES_REQUESTED** [review]

File: .github/workflows/pr_e2e_tests.yml (hunk 1 above)
Comment (JonathanConn) [reply .github/workflows/pr_e2e_tests.yml:2]: I did not ignore your pervious feedback
This draft pr was never meant to be reviewed
I was using this PR to test solutions to an issue I linked in our slack
I agree with the changes you suggested and they will be there in the final version

File: .github/workflows/pr_e2e_tests.yml (hunk 2 above)
Comment (JonathanConn) [reply .github/workflows/pr_e2e_tests.yml:6]: The previous iteration did not exist, I never wrote a previous iteration
That was auto generated by playwright
I later removed it to try and fix the confusion

**Pull request closed without merging**